Userland/Applications/SoundPlayer/main.cpp

```diff
@@ -61,10 +63,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto playback_menu = TRY(window->try_add_menu("&Playback"));
     GUI::ActionGroup loop_actions;
     loop_actions.set_exclusive(true);
-    auto loop_none = TRY(GUI::Action::try_create("&No Loop", [&](auto&) {
+    auto loop_none = GUI::Action::create_checkable("&No Loop", { Mod_Ctrl, Key_N }, [&](auto&) {
```

Comment:
Is there no `try_create_checkable`?

Reply:
I don't see see one: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibGUI/Action.h#L69-L72

But then again I don't see the old `try_create` either?

When I grep the repo, this case that I'm removing is the only match I see:
```
[*master] maxtrussell:~/dev/serenity
$ rg Action::try_create
Userland/Applications/SoundPlayer/main.cpp
64:    auto loop_none = TRY(GUI::Action::try_create("&No Loop", [&](auto&) {
```

I feel like I'm missing something, because this never should have worked?

Reply:
@awesomekling any ideas, I'm kinda lost?

Reply:
The API just doesn't exist yet. That shouldn't block this PR though.

Reply:
To answer the other question, it is currently using the `try_create` function that is added via then `C_OBJECT(...)` macro.